tsconfig.json

```diff
@@ -10,14 +10,16 @@
 		"moduleResolution": "Node",
 		"noLib": true,
 		"resolveJsonModule": true,
+		"experimentalDecorators": true,
+		"forceConsistentCasingInFileNames": true,
+		"moduleDetection": "force",
```

Comment:
Are these necessary?

Reply:
It matches up with the default `tsconfig.json` when creating a new package project with `rbxtsc init`. I don't think this is really important, though.

Reply:
Alright, those might as well stay